src/elements/content-sidebar/activity-feed/annotation-thread/AnnotationThreadContent.js

```diff
@@ -40,31 +42,18 @@ const AnnotationThreadContent = ({
         replies,
         isLoading,
         error,
-        handleEdit,
-        handleStatusChange,
-        handleDelete,
-        handleCreateReply,
-        handleDeleteReply,
-        handleEditReply,
-    } = useAnnotationAPI({
+        annotationActions: { handleAnnotationStatusChange, handleAnnotationDelete, handleAnnotationEdit },
+        repliesActions: { handleReplyEdit, handleReplyCreate, handleReplyDelete },
+    } = useAnnotationThread({
```

Comment:
We should reconsider this approach in the future, when time permits. It seems like the underlying state should live in the consuming application rather than in the component.